javascript/README.md

```diff
@@ -0,0 +1,332 @@
+# Care Pet ScyllaDB IoT example
+
+This example project demonstrates a generic IoT use case for ScyllaDB in JavaScript.
+
+The documentation for this application and guided exercise is [here](../docs).
+
+The application allows tracking of pets health indicators and consist of three parts:
+
+- migrate (`/src/cmd/migrate/index.js`) - creates the `carepet` keyspace and tables
+- collar (`/src/cmd/sensor/index.js`) - generates a pet health data and pushes it into the storage
+- web app (`/src/index.js`) - REST API service for tracking pets health state
+
+## Quick Start
+
+Prerequisites:
+
+- [NodeJS](https://nodejs.org/en/) tested with v17.0.1
+- [NPM](https://www.npmjs.com/) tested with v8.1.0
+- [docker](https://www.docker.com/)
+- [docker-compose](https://docs.docker.com/compose/)
+
+Make sure to install all NodeJS dependencies with:
+
+    $ npm install
+
+To run a local ScyllaDB cluster consisting of three nodes with
+the help of `docker` and `docker-compose` execute:
+
+    $ docker-compose up -d
+
+Docker-compose will spin up three nodes: `carepet-scylla1`, `carepet-scylla2`
+and `carepet-scylla3`. You can access them with the `docker` command.
+
+To execute CQLSH:
+
+    $ docker exec -it carepet-scylla1 cqlsh
+
+To execute nodetool:
+
+    $ docker exec -it carepet-scylla1 nodetool status
```

Comment:
Can we move this command up and show the expected result before moving forward? @guy9 what do you think?

Reply:
Do you mean moving it to be before the CQLSH command?